trio/tests/test_highlevel_open_tcp_listeners.py

```diff
@@ -81,7 +81,9 @@
             if len(client_streams) >= limit:  # pragma: no cover
                 return "lots"
     finally:
-        for client_stream in client_streams:
+        # The need for "no cover" here is subtle: see
+        # https://github.com/python-trio/trio/issues/522
+        for client_stream in client_streams:  # pragma: no cover
```

Comment:
You might prefer `#pragma: no branch`, which indicates that the line is expected to be covered, but that one of its possible branches will be missing.